Gemfile

```diff
@@ -44,7 +33,7 @@ if RUBY_ENGINE == "ruby" and RUBY_VERSION > '1.9.2'
   gem 'puma'
   gem 'net-http-server'
   gem 'yajl-ruby'
-  gem 'nokogiri'
+  gem 'nokogiri', '~> 1.5'
```

Comment:
Try pinning to nokogiri 1.6.8.1. Versions after that require Ruby 2.1.0 or greater.